### PR TITLE
Successful application submission page

### LINKED ISSUE #253: Build Domain Submission Landing Page
**User Story**

As a user submitting a domain request I want to know my domain was successfully submitted so I can prepare for future steps

**Acceptance Criteria**

- Domain Request form has a landing page that aligns with wireframes

**Wireframe**

https://www.figma.com/proto/ninNjo0bTg7XfB4mvTp4hG/GetGov%2FWireframes?page-id=1057%3A4846&node-id=1077%3A7610&viewport=-1004%2C292%2C0.15&scaling=min-zoom&starting-point-node-id=1057%3A5291

### PR DESCRIPTION
**Pull request by neilmb**
# Successful application submitted page #

## 🗣 Description ##

This adds the page from our wireframes that users see after they successfully complete the new domain request form.

Screen Shot 2022-11-21 at 10 53 49

It links to two currently non-existent pages, one a detail view for the newly completed application at `/application/<id>` and the other a dashboard at `/domains`.

## 💭 Motivation and context ##

Closes #253

### CODE REVIEW

File: src/registrar/templates/application_done.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+<ul>
+
+    <li> We'll review your request. This could take up to two weeks. During
+        this review we'll verify that you're eligible for a .gov domain, that
+        your authorizing official approves your request, and that your domain
+        meets our naming requirements.
+    </li>
+
+    <li> You can <a href="/application/{{ application_id }}">check the
```
Comment (SSPJ) [inline src/registrar/templates/application_done.html:22]: Oh, I see what you mean with `/application/id` now.

**Suggestion:** Wrap the `<a>` tags in `<del>` so we have a visual indicator that these are broken/WIP links.

**Review by SSPJ — APPROVED** [review]
Very nice page, Neil! LGTM 👍 

Consider adding to pa11y file before merging.

The Basics
- [x] I read the code
- [x] I ran the code
 
Documentation
- [x] README (or other documentation) is correct and updated
- [x] all new functions and methods are commented
    - [x] using plain language
- N/A licenses for new dependencies are recorded, if any
- N/A licenses for updated dependencies have been checked for accuracy, if any
- N/A new components have been added to the system diagram
 
Security
- N/A security false positives are documented
- N/A data from external sources is cleaned and clearly marked
- N/A significant user actions are logged to a database table
 
Reliability
- [x] good level of error handling
    - N/A interactions with external systems are wrapped in try/except
- [x] functionality is tested with unit or integration tests

Accessibility
- ❌  New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Meets WCAG 2.0 AA or 2.1 AA for Section 508 compliance
    - [x] Site is keyboard accessible. All interactions can be accessed with a keyboard
    - [x] Site is free of keyboard traps. The keyboard focus is never trapped in a loop
    - N/A All form inputs have explicit labels
    - N/A All relevant images use an img tag
    - N/A All images have alt attributes
    - N/A Multimedia is tagged. All multimedia has appropriate captioning and audio description
    - N/A Text has sufficient color contrast. All text has a contrast ratio of 4.5:1 with the background
    - N/A Site never loses focus. Focus is always visible when moving through the page with the keyboard
    - [x] Tab order is logical
    - N/A Form instructions are associated with inputs
    - N/A Site doesn’t timeout unexpectedly. Identify elements that may “timeout” and verify that the user can request more time
    - N/A Tables are coded properly. Tables have proper headers and column attributes
    - [x] Headings are nested properly. Heading elements are nested in a logical way
    - N/A Frames are named. All frames have a name element
    - N/A Flashing elements are compliant. Elements that flash on screen do so at a rate of less than 3 Hz
    - [x] Language is set. The language for the page is set
    - N/A The language for sections on the page that differ from the site language are set
    - [x] CSS is not required to use the page. The page makes sense with or without CSS
    - [x] Links are unique and contextual. All links can be understood taken alone, e.g., ‘Read more - about 508’
    - [x] Page titles are descriptive
    - N/A Required plugins are linked on the page
 
Device Matrix
- [x] firefox
    - [x] web pages render correctly
    - [x] interactive functionality is operative
- [x] chrome/chromium/edge
    - [x] web pages render correctly
    - [x] interactive functionality is operative
- ❌  safari: a bug is preventing testing at the moment
- [x] web page is readable and usable 
    - [x] at 480px
    - [x] at 1024px

File: src/registrar/templates/application_done.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+</ul>
+
+<h2>Option to enter domain name server information</h2>
+
+<p>Before your domain can be used we'll need information about your
+domain name servers. If you have this information you can enter it now.
+If you don't have it, that's okay. You can enter it later on the
+<a href="/domains">manage your domains page</a>.
+</p>
+{% endblock %}
```
Comment (igorkorenfeld) [inline src/registrar/templates/application_done.html:37]: One small item left out is to a placeholder button "Enter DNS name servers" 
image

**Review by igorkorenfeld — CHANGES_REQUESTED** [review]
This is looking great Neil! Just one small things left out

File: src/registrar/templates/application_done.html (hunk 2 above)
Comment (neilmb) [reply src/registrar/templates/application_done.html:37]: Added, thanks for catching that:
Screen Shot 2022-11-22 at 11 28 17

**Review by igorkorenfeld — APPROVED** [review]
Thanks for adding it back in, this is looking great

**Pull request merged**